src/lib/netlist/plib/prandom.h

```diff
@@ -190,43 +196,43 @@ namespace plib
 				FT v2;
 				do
 				{
-					v1 = normalize_uniform(p, constants<FT>::two(), constants<FT>::one()); // [-1..1[
-					v2 = normalize_uniform(p, constants<FT>::two(), constants<FT>::one()); // [-1..1[
+					v1 = normalize_uniform(p, constants<FT>::two(),
+										   constants<FT>::one()); // [-1..1[
+					v2 = normalize_uniform(p, constants<FT>::two(),
+										   constants<FT>::one()); // [-1..1[
 					s = v1 * v1 + v2 * v2;
 				} while (s >= constants<FT>::one());
 				if (s == constants<FT>::zero())
 				{
 					m_buf[i] = s;
-					m_buf[i+1] = s;
+					m_buf[i + 1] = s;
 				}
 				else
 				{
 					// last value without error for log(s)/s
 					// double: 1.000000e-305
 					// float: 9.999999e-37
-					// FIXME: with 128 bit randoms log(s)/w will fail 1/(2^128) ~ 2.9e-39
-					const FT m(m_stddev * plib::sqrt(-constants<FT>::two() * plib::log(s)/s));
+					// FIXME: with 128 bit randoms log(s)/w will fail 1/(2^128)
+					// ~ 2.9e-39
+					const FT m(
+						m_stddev
+						* plib::sqrt(-constants<FT>::two() * plib::log(s) / s));
 					m_buf[i] = /*mean+*/ m * v1;
-					m_buf[i+1] = /*mean+*/ m * v2;
+					m_buf[i + 1] = /*mean+*/ m * v2;
 				}
 			}
 			m_p = 0;
 		}
 
 		std::array<FT, 256> m_buf;
-		std::size_t m_p;
-		FT m_stddev;
+		std::size_t         m_p;
+		FT                  m_stddev;
 	};
 
 	using mt19937_64 = mersenne_twister_t<
-		uint_fast64_t,
-		64, 312, 156, 31,
-		0xb5026f5aa96619e9ULL,
-		29, 0x5555555555555555ULL,
-		17, 0x71d67fffeda60000ULL,
-		37, 0xfff7eee000000000ULL,
-		43,
-		6364136223846793005ULL>;
+		uint_fast64_t, 64, 312, 156, 31, 0xb5026f5aa96619e9ULL, 29,
+		0x5555555555555555ULL, 17, 0x71d67fffeda60000ULL, 37,
+		0xfff7eee000000000ULL, 43, 6364136223846793005ULL>;
```

Comment:
These were logically grouped before – now they’re randomly grouped.  This makes the code harder to understand.